.gitignore

```diff
@@ -7,4 +7,5 @@
 build/
 ios/.generated/
 ios/Flutter/Generated.xcconfig
+ios/Flutter/flutter_export_environment.sh
```

Comment:
This file should not be in vcs.
I didn't delete it from the vcs because it would probably cause you trouble, so you're welcome to remove it:
```bourne shell
$ git rm --cached ios/Flutter/flutter_export_environment.sh
```